docs/pt/docs/tutorial/bigger-applications.md

```diff
@@ -0,0 +1,556 @@
+
+/// info
+
+Se você vem do Flask, isso seria o equivalente aos Blueprints do Flask.
+
+///
+
+## Um exemplo de estrutura de arquivos
+
+Digamos que você tenha uma estrutura de arquivos como esta:
+
+```
+.
+├── app
+│   ├── __init__.py
+│   ├── main.py
+│   ├── dependencies.py
+│   └── routers
+│   │   ├── __init__.py
+│   │   ├── items.py
+│   │   └── users.py
+│   └── internal
+│       ├── __init__.py
+│       └── admin.py
+```
+
+/// tip | "Dica"
+
+Existem vários arquivos `__init__.py` presentes em cada diretório ou subdiretório.
+
+Isso permite a importação de código de um arquivo para outro.
+
+Por exemplo, no arquivo `app/main.py`, você poderia ter uma linha como:
+
+```
+from app.routers import items
+```
+
+///
+
+* O diretório `app` contém todo o código da aplicação. Ele possui um arquivo `app/__init__.py` vazio, o que o torna um "pacote Python" (uma coleção de "módulos Python"): `app`.
+* Dentro dele, o arquivo `app/main.py` está localizado em um pacote Python (diretório com  `__init__.py`). Portanto, ele é um "módulo" desse pacote: `app.main`.
+* Existem também um arquivo `app/dependencies.py`, assim como o `app/main.py`, ele é um "módulo": `app.dependencies`.
+* Há um subdiretório `app/routers/` com outro arquivo `__init__.py`, então ele é um "subpacote Python": `app.routers`.
+* O arquivo `app/routers/items.py` está dentro de um pacote, `app/routers/`, portanto, é um "submódulo": `app.routers.items`.
+* O mesmo com `app/routers/users.py`,  ele é outro submódulo: `app.routers.users`.
+* Há também um subdiretório `app/internal/` com outro arquivo `__init__.py`, então ele é outro "subpacote Python":`app.internal`.
+* E o arquivo `app/internal/admin.py` é outro submódulo: `app.internal.admin`.
+
+<img src="/img/tutorial/bigger-applications/package.svg">
+
+A mesma estrutura de arquivos com comentários:
+
+```
+.
+├── app                  # "app" é um pacote Python
+│   ├── __init__.py      # este arquivo torna "app" um "pacote Python"
+│   ├── main.py          # "main" módulo, e.g. import app.main
+│   ├── dependencies.py  # "dependencies" módulo, e.g. import app.dependencies
+│   └── routers          # "routers" é um  "subpacote Python"
+│   │   ├── __init__.py  # torna "routers" um "subpacote Python"
+│   │   ├── items.py     # "items" submódulo, e.g. import app.routers.items
+│   │   └── users.py     # "users" submódulo, e.g. import app.routers.users
+│   └── internal         # "internal" é um  "subpacote Python"
+│       ├── __init__.py  # torna "internal" um  "subpacote Python"
+│       └── admin.py     # "admin" submódulo, e.g. import app.internal.admin
+```
+
+## `APIRouter`
+
+Vamos supor que o arquivo dedicado a lidar apenas com usuários seja o submódulo em `/app/routers/users.py`.
+
+Você quer manter as *operações de rota* relacionadas aos seus usuários separadas do restante do código, para mantê-lo organizado.
+
+Mas ele ainda faz parte da mesma **FastAPI** aplicação/web API (faz parte do mesmo "pacote Python").
```

Comment:
I think it makes more sense to put FastAPI after *aplicação/web API*:

`Mas ele ainda faz parte da mesma aplicação/web API **FastAPI** `